Chessie.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 2013
```

Comment:
This is the part I wasn't sure of. I only have VS2013 and 2015 on this box. 
VS forced the change when I added the test project to the solution, kept as a separate commit.